oozie-to-airflow/run-sys-test

```diff
@@ -361,6 +365,15 @@ function prepare_configuration {
     fi
 }
 
+function prepare_gcloud_configuration {
```

Comment:
Similarly as in other functions - please add header and footer showing what's going on. Again for diagnostics it is very important for the user to see what's going on.

Reply:
I updated a PR.